### DISCUSSION TITLE
Native Image  Unable to create build io.quarkus.builder.BuildException:

### DISCUSSION THREAD

**Discussion by techie-stacks** [quarkusio/quarkus#23910] [Q&A]
****Failed to build quarkus application: io.quarkus.builder.BuildException: Build failure: Build failed due to errors****

**Comment by techie-stacks** [comment]
Caused by: java.lang.RuntimeException: Image generation failed. Exit code: 1
[ERROR] 	at io.quarkus.deployment.pkg.steps.NativeImageBuildStep.imageGenerationFailed(NativeImageBuildStep.java:365)
[ERROR] 	at io.quarkus.deployment.pkg.steps.NativeImageBuildStep.build(NativeImageBuildStep.java:200)
[ERROR] 	... 11 more

**Comment by techie-stacks** [comment]
Using maven 3.8.3 and gralvm 21.1

**Comment by gsmet** [comment]
`io.quarkus:quarkus-maven-plugin:2.0.0.Final:build` is weird. You were referencing a much newer version of Quarkus in your initial message. The version used for the Maven plugin and the BOM have to be consistent. Try to use 2.7.2.Final for both as a start.

Then the native-image binary outputs something when it builds the image. If we don't have that, we can't help you.
